specs/_features/eip7594/das-core.md

```diff
@@ -106,18 +106,20 @@ def get_custody_columns(node_id: NodeID, custody_subnet_count: uint64) -> Sequen
     assert custody_subnet_count <= DATA_COLUMN_SIDECAR_SUBNET_COUNT
 
     subnet_ids: List[uint64] = []
-    i = 0
+    tmp_id = uint256(node_id)
     while len(subnet_ids) < custody_subnet_count:
-        if node_id == UINT256_MAX:
-            node_id = NodeID(0)
+        # Overflow prevention
+        if tmp_id == UINT256_MAX:
```

Comment:
It makes more sense to move this part of the code below, just before the increment like @dankrad did in #3752, so we will not skip uint256_max iteration step